langchain4j-jina-ai/src/main/java/dev/langchain4j/model/jinaAi/JinaAiClient.java

```diff
@@ -0,0 +1,67 @@
+package dev.langchain4j.model.jinaAi;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import lombok.Builder;
+import okhttp3.OkHttpClient;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+import java.io.IOException;
+import java.time.Duration;
+
+import static com.google.gson.FieldNamingPolicy.LOWER_CASE_WITH_UNDERSCORES;
+import static dev.langchain4j.internal.ValidationUtils.ensureNotBlank;
+
+public class JinaAiClient {
+    private static final Gson GSON = new GsonBuilder()
```

Comment:
optional: it would be great to use Jackson instead, as we are starting to get rid of Gson. More details: https://github.com/langchain4j/langchain4j/pull/1043